ext/ftp/ftp.stub.php

```diff
@@ -2,7 +2,7 @@
 
 /** @generate-class-entries */
 
-namespace FTP {
+namespace {
```

Comment:
Can you please add a test inside ext/zend_test?

Reply:
Added a simple test, which I think will do for this change.